rpcd/playbooks/roles/rpc_maas/templates/cinder_vg_check.yaml.j2

```diff
@@ -0,0 +1,20 @@
+type: agent.plugin
+label: cinder_vg_check--{{ ansible_hostname }}
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : vg_check.py
+    args    : ["{{ item.cinder_vg_name }}"]
+alarms      :
+    cinder_vg_space_status :
+        label                   : cinder_vg_space_status--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (percentage(metric["{{ item.cinder_vg_name }}_vg_used_space"], metric["{{ item.cinder_vg_name }}_vg_total_space"]) > {{ cinder_volumes_vg_warning_threshold }}) {
+                return new AlarmStatus(WARNING, "space in the {{ item.cinder_vg_name }} vg on this host is {{ cinder_volumes_vg_warning_threshold }}% used");
```

Comment:
I think message should be adjusted e.g. 

"Space in the {{ item.cinder_vg_name }} vg on this host is at least {{ cinder_volumes_vg_warning_threshold }}% used"